analyses/2009_03_noahwalugembe__Internet-Jones-evolution.txt

```diff
@@ -0,0 +1,32 @@
+Verification / evolution of "Internet Jones" paper #26
+Buy
+Walugembe Francis Noah
+noahwalugembe@gmail.com
+
+Introduction
+
+Third-party web tracking is the practice by which entities (“trackers”) embedded in webpages re-identify users as they browse the web, collecting information about the websites that they visit. A cording to According to  Lerner, Simpson, Kohno and Roesner, (2016) web Tracking is typically done for the purposes of website analytics, targeted advertising, and other forms of personalization (e.g., social media content). In this work I am evaluating the contribution of "Internet Jones" paper #26 starting with its insight on TrackingExcavator and a longitudinal measurement study of third-party cookie-based web tracking on Wayback Machine1. I will also show how has the third-party web tracking ecosystem evolved since its beginnings according to "Internet Jones" paper.
```

Comment:
This is a great introduction and I appreciate you clearly laying out background and the goal of your analysis.

> `In this work I am evaluating the contribution of "Internet Jones" paper #26 starting with...`

One small thing: `#26` refers to the issue not the paper - joining the two together like this doesn't make sense. You could omit the `#26` so it reads `In this work I am evaluating the contribution of "Internet Jones" paper starting with...`

One bigger thing: To evaluate the Internet Jones paper was not the intention of #26. The goal of #26 was to apply applicable methodologies from the Internet Jones paper to the OverScripted dataset to compare the results we see with the results that were presented in the Internet Jones paper.